components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/constants/RoleMgtConstants.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.role.v2.mgt.core.constants;
+
+/**
+ * Constants for role management.
+ */
+public class RoleMgtConstants {
```

Comment:
Why can't we use `org.wso2.carbon.identity.role.v2.mgt.core.RoleConstants` instead of creating this new class?

Reply:
Addressed and resolved in commit: [42705b9](https://github.com/wso2/carbon-identity-framework/pull/6162/commits/42705b984942a14ec734891444d31c9d33397a6f)